File: MedLabO.Models/Usluga.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MedLabO.Models
{
    public class Usluga
    {
        public string UslugaID { get; set; }
        public string Naziv { get; set; }
        public string Opis { get; set; }
        public decimal Cijena { get; set; }
        public int TrajanjeUMin { get; set; }
        public float RezultatUH { get; set; }
        public bool Dostupno { get; set; }
        public DateTime DTKreiranja { get; set; }
        public DateTime? DTZadnjeModifikacije { get; set; }
        public byte[]? Slika { get; set; }
        public ICollection<Test.Test> UslugaTestovi { get; set; } = new List<Test.Test>();
        public string? AdministratorID { get; set; }
        public Administrator? Administrator { get; set; }
    }
}

File: MedLabO UI/medlabo_mobile/lib/models/usluga/usluga.dart
import 'package:json_annotation/json_annotation.dart';
import 'package:medlabo_mobile/models/test/test.dart';

part 'usluga.g.dart';

@JsonSerializable()
class Usluga {
  int? uslugaID;
  String? naziv;
  String? opis;
  double? cijena;
  int? trajanjeUMin;
  double? rezultatUH;
  bool? dostupno;
  String? dtKreiranja;
  String? dtZadnjeModifikacije;
  String? slika;
  List<Test>? uslugaTestovi;
  String? administratorID;

  Usluga(
      {this.uslugaID,
      this.naziv,
      this.opis,
      this.cijena,
      this.trajanjeUMin,
      this.rezultatUH,
      this.dostupno,
      this.dtKreiranja,
      this.dtZadnjeModifikacije,
      this.slika,
      this.uslugaTestovi,
      this.administratorID});

  factory Usluga.fromJson(Map<String, dynamic> json) => _$UslugaFromJson(json);

  Map<String, dynamic> toJson() => _$UslugaToJson(this);
}
